python/lsst/ip/diffim/dipoleFitTask.py

```diff
@@ -748,6 +750,11 @@ def dipoleModelFunctor(x, flux, xcenPos, ycenPos, xcenNeg, ycenNeg, fluxNeg=None
         if np.any(~mask):
             weights[~mask] = 0.
 
+        # Filter out any nans, and make the weights 0.
+        nans = (~np.isfinite(z) | ~np.isfinite(weights))
+        z[nans] = 0
```

Comment:
I try to avoid setting image values to zero, in case there is a non-zero offset. Perhaps `z[nans] = np.nanmedian(z)`